dtfs-central-api/src/v1/routes/portal-routes.js

```diff
@@ -59,6 +59,8 @@ portalRouter.use((req, res, next) => {
  *                 properties:
  *                   details:
  *                     type: object
+ *               auditDetails:
```

Comment:
Suprised to not see lots of api-tests calling this endpoint updated (e.g. dtfs-central-api\api-tests\v1\deals\deal.api-test.js)